hooks/post/30-upgrade.rb

```diff
@@ -12,7 +17,11 @@ def fail_and_exit(message)
 
 if app_value(:upgrade)
   if [0, 2].include?(@kafo.exit_code)
-    upgrade_tasks if Kafo::Helpers.module_enabled?(@kafo, 'foreman')
+    if module_enabled?('foreman')
```

Comment:
Does that work in 3.10 already?

Reply:
Good point. It doesn't work as it should since it errors out if the module doesn't exist (like in the foreman-proxy-content scenario).